lib/main_containers.pm

```diff
@@ -155,6 +155,20 @@ sub load_host_tests_helm {
     }
 }
 
+sub load_image_tests_in_k8s {
+    my ($run_args) = @_;
+    my $providers = undef;
+
+    $providers = get_var("PUBLIC_CLOUD_PROVIDER", "GCE,EC2,AZURE");
+
+    foreach (split(',\s*', $providers)) {
+        push @{$run_args->{provider}}, $_;
+        loadtest('containers/push_container_image_to_pc', run_args => $run_args, name => "push_container_image_to_" . $_);
+        push @{$run_args->{provider}}, $_;
+        loadtest('containers/run_container_in_k8s', run_args => $run_args, name => "run_container_in_k8s_" . $_);
```

Comment:
Why is the `push` called twice?

Reply:
We need to queue twice because we need to extract this twice (from the two tests). In each test one element of this queue is extracted